conf/application.conf

```diff
@@ -197,4 +197,397 @@ versioning {
+                "BE",
+                "BF",
+                "BG",
+                "BH",
+                "BI",
+                "BJ",
+                "BO",
+                "BP"
+            ]
+        }
+        "read:individuals-income-laa-c3" {
+            fields = [
+                "A",
+                "B",
+                "C",
+                "D",
+                "E",
+                "F",
+                "J",
+                "K",
+                "L",
+                "M",
+                "N"
+                "O",
+                "P",
+                "Q"
+                "S",
+                "T",
+                "U",
+                "V",
+                "X",
+                "Y",
+                "Z",
+                "AC",
+                "AD",
+                "AE",
+                "AF",
+                "AG",
+                "AH",
+                "AK",
+                "AL",
+                "AM",
+                "AN",
+                "AP",
+                "AQ",
+                "AR",
+                "AS",
+                "AT",
+                "AU",
+                "AV",
+                "AW",
+                "AX",
+                "AY",
+                "AZ",
+                "BA",
+                "BB",
+                "BC",
+                "BD",
+                "BE",
+                "BF",
+                "BG",
+                "BH",
+                "BI",
+                "BJ",
+                "BP",
+                "BQ",
+                "BR",
+                "BS",
+                "BT",
+                "BU",
+                "BV",
+                "BW",
+                "BX"
+            ]
+        }
+        "read:individuals-income-laa-c4" {
+            fields = [
+                "A",
+                "B",
+                "C",
+                "D",
+                "E",
+                "F",
+                "G",
+                "J",
+                "AJ",
+                "AP",
+                "BC",
+                "BD",
+                "BE",
+                "BF"]
+        }
+
+        "read:individuals-income-hmcts-c2" {
+            fields = ["H", "J", "K", "M", "N", "O", "R", "T", "U", "AI", "AP", "BC", "BD", "BE", "BF", "BG"]
+        }
+
+        "read:individuals-income-hmcts-c3" {
+            fields = ["H", "J", "K", "M", "N", "O", "R", "T", "U", "AI", "AP", "BC", "BD", "BE", "BF", "BG"]
+        }
+
+        "read:individuals-income-hmcts-c4" {
+            fields = ["A", "B", "C", "D", "E", "F", "G", "H", "J", "AJ", "AO", "AP", "BC", "BD", "BE", "BF"]
+        }
+
+        "read:individuals-income-lsani-c1" {
+            fields = ["I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "AA", "AB", "AF", "AH", "AI", "AL", "AM", "AN", "AP", "AQ", "BQ", "BR", "BS", "BT", "BU", "BV", "BW", "BX", "AR", "AS", "AT", "AU", "AV", "AW", "AX", "AY", "AZ", "BA", "BB", "BC", "BD", "BE", "BF", "BG", "BH", "BI", "BJ", "BK", "BL", "BM", "BN", "BO", "BP"
```

Comment:
Why separate the fields onto newlines at the top of the file and then go to one line here? If length > 80 chars then drop to newlines? Maybe 10 per line? Just play about with the best option for maintainability.

Reply:
Seperated into chunks of 10. Reads a lot better, good shout.